packages/worker/native/src/lib.rs

```diff
@@ -8,7 +8,7 @@ use temporal_sdk_core::{
     init, protos::coresdk::workflow_completion::WfActivationCompletion,
     protos::coresdk::ActivityHeartbeat, protos::coresdk::ActivityTaskCompletion, tracing_init,
     ClientTlsConfig, CompleteActivityError, CompleteWfError, Core, CoreInitError, CoreInitOptions,
-    PollActivityError, PollWfError, ServerGatewayOptions, TlsConfig, Url,
+    PollActivityError, PollWfError, ServerGatewayOptions, TlsConfig, Url, WorkerConfig,
```

Comment:
I just your pipeline doesn't run clippy which would be a nice thing to do. Probably there's a handful of suggestions in here that'd be useful. You can check out the buildkite definition in Core for how we run clippy in pipelines.

Reply:
Sure, I'll open an issue.